modules/disabling-redirect-private-storage-endpoint-azure.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-private-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="disabling-redirect-private-storage-endpoint-azure"]
+= Optional: Disabling redirect when using a private storage endpoint on Azure
+
+By default, redirect is enabled when using the image registry with a private storage endpoint on Azure. This ensures that users within the same cluster can pull images from the registry. 
+
+When redirect is enabled, users from outside of the cluster are unable to pull images from the registry. In some cases, users might want to disable redirect so that users from outside of the cluster can pull images from the registry. 
+
+Use the following procedure to disable redirect.
+
+.Prerequisites
+
+* You have configured an image registry to run on Azure
+
+.Procedure
+
+* Enter the following command to disable redirect on the image 
+registry configuration:
++
+[source,terminal]
+----
+$ oc patch configs.imageregistry cluster --type=merge -p '{"spec":{"disableRedirect": true}}'
+----
+
+.Verification
+
+[NOTE]
+====
+The following procedure requires proper verification from a user outside of the cluster.
+====
+
+. Fetch the registry endpoint by running the following command:
++
+[source,terminal]
+----
+$ REGISTRY=$(oc get route default-route -n openshift-image-registry --template='{{ .spec.host }}')
```

Comment:
what if the user haven't configured a route? should a route be a prerequisite? or do we just note that somewhere?

Reply:
no a route is not a prerequisite for making storage account private.